### PR TITLE
[#614] Use `leaflet-tools`

### LINKED ISSUE #614: Use leaflet-tools package
In favour of the copy-pasted rd.js/map.js code.

### PR DESCRIPTION
**Pull request by Viicos**
Fixes #614

### CODE REVIEW

**Comment by Viicos** [comment]
Hitting a weird test failure. Might be related: https://github.com/Leaflet/Leaflet/issues/6416#issuecomment-1360369989

File: src/map/constants.js
Comment (sergei-maertens) [inline src/map/constants.js:1]: I was hoping we could get rid of this file entirely and instead use only the exports from `@open-formulieren/leaflet-tools`

**Comment by sergei-maertens** [comment]
> Hitting a weird test failure. Might be related: [Leaflet/Leaflet#6416 (comment)](https://github.com/Leaflet/Leaflet/issues/6416#issuecomment-1360369989)

Try updating leaflet to the latest version, this code works in the new form builder :thinking:

File: src/map/constants.js
Comment (Viicos) [reply src/map/constants.js:1]: We could but there are still some constants specific to the sdk: `DEFAULT_LAT_LNG`, `DEFAULT_ZOOM`

**Comment by Viicos** [comment]
> Try updating leaflet to the latest version, this code works in the new form builder 🤔

tried with no eval

**Review by SilviaAmAm — APPROVED** [review]

**Comment by Viicos** [comment]
Regarding sergei's questions: he asked if it was possible to get rid of the `constants` module, but the current form builder does not set default values for the center. So did a bit of cleanup but still keeping the module

**Pull request merged**